Source/Scene/ModelExperimental/CustomShaderGuide/README.md

```diff
@@ -202,7 +202,7 @@ An automatically-generated GLSL struct that contains attributes.
 // this struct represents the raw attribute values.
```

Comment:
The custom shader receives access to the processed attributes values now, right?

It would be worthwhile to provide the full list of processed attributes since there are some special cases that should be pointed out to the user.

* `tangent` - is a vec3
* `color_0`, `color_1`, etc - are vec4

Any others? I forget, is bitangent exposed to the custom shader, or is it just used internally to compute the normal from the normal map?

Make sure to do a careful pass over the whole document to update other areas that may have changed.

Reply:
Side question - are `positionMC`, `positionEC`, `positionWC` part of the `Attributes` struct or part of the `FragmentInput` struct or both?

I see that the the third sandcastle has a custom shader that does `fsInput.positionMC` but the documentation here shows `positionMC` in `Attributes`.